enterprise/web.jsf/src/org/netbeans/modules/web/jsf/wizards/JSFConfigurationPanelVisual.java

```diff
@@ -153,15 +152,13 @@ public class JSFConfigurationPanelVisual extends javax.swing.JPanel implements H
     private static volatile boolean jsfLibrariesCacheDirty = true;
     private static final List<Library> JSF_LIBRARIES_CACHE = new CopyOnWriteArrayList<>();
 
-    /** Map used for faster seek of JSF registered libraries. */
-    private static final Map<Boolean, String> JSF_SEEKING_MAP = new LinkedHashMap<>(2);
-
-    static {
-        JSF_SEEKING_MAP.put(false, JSFUtils.EJB_STATELESS);    //NOI18N
-        JSF_SEEKING_MAP.put(true, JSFUtils.FACES_EXCEPTION);   //NOI18N
-        JSF_SEEKING_MAP.put(false, JSFUtils.JAKARTAEE_EJB_STATELESS);    //NOI18N
-        JSF_SEEKING_MAP.put(true, JSFUtils.JAKARTAEE_FACES_EXCEPTION);   //NOI18N
-    }
+    /** Maps used for faster seek of JSF/Jakarta Faces registered libraries. */
+    private static final Map<Boolean, String> JSF_SEEKING_MAP = Map.of(
+            false, JSFUtils.EJB_STATELESS,
+            true, JSFUtils.FACES_EXCEPTION);
```

Comment:
This change is invalid. `Map#of` explicitly returns a map with undefined iteration order:

> The iteration order of mappings is unspecified and is subject to change. 

while `org.netbeans.modules.j2ee.common.ClasspathUtil.containsClass(List<URL>, Map<T, String>)` explictly states:

> The value returned is the key of corresponding classname value in the classNames map. The priority of the classnames is determined by the Map iterator, so if the key values to be returned are distinct the Map with defined iteration should be used (such as [TreeMap](https://github.com/apache/netbeans/pull/8623/*9) or [LinkedHashMap](https://github.com/apache/netbeans/pull/8623/*10)).

So the split looks correct, but the implementation needs to be adjusted.